Documentation/gettingstarted/vtep.rst

```diff
@@ -118,14 +118,7 @@ run the commands below using ``sudo``.
    ip addr add 10.1.1.236/24 dev vxlan2
    # Assume Cilium podCIDR network is 10.0.0.0/16, add route to 10.0.0.0/16
    ip route add 10.0.0.0/16 dev vxlan2  proto kernel  scope link  src 10.1.1.236
-   # Allow Linux VM send ARP broadcast request to Cilium node for busybox pod ARP resolution
-   # through vxlan2 device, note this depend on if Cilium is able to proxy busybox pod ARP
```

Comment:
Should the first part of this comment remain, to explain what the `bridge` command below is for?